app/controllers/renderLogin.js

```diff
@@ -1,5 +1,12 @@
-function renderLogin(req, res,) {
-    res.status(200).render("login", {error: false});
+require("dotenv").config();
+
+function renderLogin(req, res, ) {
+  const authLink = 'https://untappd.com/oauth/authenticate/?client_id=' + process.env.CLIENTID + '&response_type=code&redirect_url=' + process.env.REDIRECT_URL;
```

Comment:
See what you do at `searchBeerHome.js` with the link, I think you can apply that here too so it's a bit more readable.